code/go/0chain.net/chaincore/chain/handler.go

```diff
@@ -1318,6 +1319,18 @@ func PutTransaction(ctx context.Context, entity datastore.Entity) (interface{},
 		return nil, err
 	}
 
+	s, err := sc.GetStateById(sc.GetLatestFinalizedBlock().ClientState, sc.OwnerID())
```

Comment:
sc.OwnerID() is not correct
it should be txn.FromClient()

Reply:
Fixed.